### DISCUSSION TITLE
How to use Binance Smart chain in Wagmi with chainId 56 or 97?

### DISCUSSION THREAD

**Discussion by ghost** [wevm/wagmi#454] [Q&A]
I've set the following:
```js
const { chains, provider } = configureChains(
  [chain.mainnet],
  [apiProvider.jsonRpc(chain => ({
    rpcUrl: `https://bsc-dataseed.binance.org/`,
  })),]
)

```

**Comment by tmm** [comment]
This looks like a question better suited for [RainbowKit](https://github.com/rainbow-me/rainbowkit). The `configureChains` API isn't a part of wagmi (yet).

**Comment by jxom** [comment]
With the introduction of `0.9.x`, wagmi now comes with the `bsc` & `bscTestnet` chains built-in (re-exported from our [`@wagmi/chains` reference repository](https://github.com/wagmi-dev/references/tree/main/packages/chains)). Read more [here](https://wagmi.sh/react/chains).
